scripts/cooper/run_mitochondria_segmentation.py

```diff
@@ -7,8 +7,13 @@
 
 def run_mitochondria_segmentation(args):
     tiling = parse_tiling(args.tile_shape, args.halo)
-    segmentation_function = partial(segment_mitochondria, model_path=args.model_path, verbose=False, tiling=tiling)
-    inference_helper(args.input_path, args.output_path, segmentation_function, force=args.force, data_ext=args.data_ext)
+    segmentation_function = partial(
+        segment_mitochondria, model_path=args.model_path, verbose=False, tiling=tiling, scale=[0.5, 0.5, 0.5]
```

Comment:
I have hard-coded the scale factor to 1/2 here. Given how much better the mito segmentation looks with this we can stick with it from now on and don't need to experiment further.